.gitignore

```diff
@@ -13,3 +13,6 @@ npm-debug.log
 # coverage
 .nyc_output
 coverage
+
+# editors
+.idea/
```

Comment:
It can be, but most people don't use this as standard .gitignores generated by github already include:

```
# Editors
.idea/*
*.iml
*.sublime-*
```

by extending the .gitignore you can avoid accidental commits, but I also can remove it again if you want me to